Extensions/xkit_preferences.js

```diff
@@ -2445,16 +2627,16 @@ XKit.extensions.xkit_preferences = new Object({
 
 		$("#xkit-open-credits").click(function() {
 
-				XKit.window.show("Credits",
-					'XKit uses <a href="www.jquery.com">jQuery and jQuery UI</a> by jQuery Foundation, '+
-					'<a href="https://github.com/timrwood/moment/">moment.js</a> by Tim Wood, '+
-					'<a href="http://code.drewwilson.com/entry/tiptip-jquery-plugin">TipTip</a> by Drew Wilson and '+
-					'<a href="http://jamesflorentino.github.io/nanoScrollerJS/">nanoScroll</a> by James Florentino. '+
-					'<br/><br/>'+
-					'XKit is written by <a href="http://www.studioxenix.com/">STUDIOXENIX</a>, a one-man entity.<br/><br/>'+
-					'All trademarks are the property of their respective owners.<br/>'+
-					'By using this software you are agreeing to <a href="http://www.xkit.info/eula">XKit EULA</a>.',
-				"info", '<div class="xkit-button default" id="xkit-close-message">OK</div>');
+			XKit.window.show("Credits",
+			'XKit uses <a href="www.jquery.com">jQuery and jQuery UI</a> by jQuery Foundation, '+
+			'<a href="https://github.com/timrwood/moment/">moment.js</a> by Tim Wood, '+
+			'<a href="http://code.drewwilson.com/entry/tiptip-jquery-plugin">TipTip</a> by Drew Wilson and '+
+			'<a href="http://jamesflorentino.github.io/nanoScrollerJS/">nanoScroll</a> by James Florentino. '+
+			'<br/><br/>'+
+			'XKit is written by <a href="http://www.studioxenix.com/">STUDIOXENIX</a>, a one-man entity.<br/><br/>'+
+			'All trademarks are the property of their respective owners.<br/>'+
+			'By using this software you are agreeing to <a href="http://www.xkit.info/eula">XKit EULA</a>.',
+			"info", '<div class="xkit-button default" id="xkit-close-message">OK</div>');
```

Comment:
For the time being, I think it's fine to use the old indentation. Increasing the diff size unnecessarily means extra things to review that could be skipped.

In the long run, we may benefit from having a style guide. For example, here are [Mozilla's](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style) and [Google's](https://github.com/google/styleguide) style guides.